src/nuxt.config.js

```diff
@@ -67,6 +68,7 @@ export default {
   ssr:            false,
   styleResources: {
     // only import functions, mixins, or variables, NEVER import full styles https://github.com/nuxt-community/style-resources-module#warning
+    hoistUseStatements: true,
     scss: [
```

Comment:
Nit: lint warning (it wants the `[` to be more indented to match the `true` in the line above).